src/components/Combobox/components/Combobox.tsx

```diff
@@ -53,6 +56,9 @@ const ComboboxRoot = ({
     allowCustomValues,
     selectedValue,
   });
+
+  const { emptyOption } = useComboboxEmptyOption();
```

Comment:
Question: What's the benefit of having this as a separate hook? If this is going to be used somewhere else then it's fine 🙂